### ISSUE TITLE
I17: Inconsistent comment

### ISSUE THREAD

**Issue by philippecamacho** [EspressoSystems/espresso-sequencer#1753]
**Affected Code:** PlonkVerifier.sol (line 769)

**Summary:** The comment referenced above has a discrepancy between the code and the comment. While the code correctly uses `verifyingKey.sigma4` , $[s_{\sigma4}]1$, the comment incorrectly refers to $[s_{\sigma3}]1$

**Suggestion:** We suggest updating the comment to reflect the formula accurately.

**Comment by philippecamacho** [comment]
I am not following:
The comment (PlonkVerifier.sol line 769) says:
```
// secondScalar = alpha * beta * z_w * [s_sigma_3]_1
//              * (wireEval0 + gamma + beta * sigmaEval0)
//              * (wireEval1 + gamma + beta * sigmaEval1)
```

It sounds that we should remove `[s_sigma_3]_1`  (instead of replacing it by `[s_sigma_4]_1`). The reason is that  `[s_sigma_3]_1` is a G_1 point  and thus cannot be part of a scalar expression.

**Comment by philippecamacho** [comment]
Closed by #1812.
